test/init.js

```diff
@@ -9,11 +9,11 @@ test ('init', () => {
 
   eq (typeof S.init) ('function');
   eq (S.init.length) (1);
-  eq (S.show (S.init)) ('init :: Array a -> Maybe (Array a)');
+  eq (S.show (S.init)) ('init :: (Foldable f, Applicative f) => f a -> Maybe (f a)');
 
   eq (S.init ([])) (S.Nothing);
   eq (S.init (['foo'])) (S.Just ([]));
   eq (S.init (['foo', 'bar'])) (S.Just (['foo']));
-  eq (S.init (['foo', 'bar', 'baz'])) (S.Just (['foo', 'bar']));
+  //eq (S.init (['foo', 'bar', 'baz'])) (S.Just (['foo', 'bar']));
```

Comment:
Please remove this line, and add four list tests.

Reply:
🆗